src/completers/git.zsh

```diff
@@ -395,7 +425,7 @@ _fzf_complete_git-unstaged-files() {
                     -v red=${fg[red]} \
                     -v reset=$reset_color '
                         '$_fzf_complete_awk_functions'
-                        /^.[^ ]/ {
+                        /'${regexes[$files_kind]}'/ {
```

Comment:
Do not interpolate a variable into the midst of an awk program. Pass the `$state` via `-v` and use that in conjunction with regex like the following:
```
state == "staged" && /^[^ ]/ {
    // ...
}
```